src/pages/WishList/WishList.css

```diff
@@ -0,0 +1,48 @@
+
+html,
+body {
+  max-width: 100%;
+  overflow-x: clip;
+}
+
+hr {
```

Comment:
avoid using style directly on element use class instead

Reply:
Okay. Will fix it later.